xlgui/properties.py

```diff
@@ -80,10 +81,10 @@ def __init__(self, parent, tracks, current_position=0, with_extras=False):
         self.__default_attributes = Pango.AttrList()
         self.__changed_attributes = Pango.AttrList()
 
-        self.message = dialogs.MessageBar(
-            parent=self.builder.get_object('main_container'),
-            buttons=Gtk.ButtonsType.CLOSE
-        )
+        main_container = self.builder.get_object('main_container')
+        self.message = dialogs.MessageBar(buttons=Gtk.ButtonsType.CLOSE)
+        main_container.insert_row(0)
+        main_container.attach(self.message, 0, 0, 6, 1)
```

Comment:
Are you sure about that? Gtk+ people have deprecated Gtk.VBox (so you probably meant a Gtk.Box with Gtk.Orientation.VERTICAL). But they also told to prefer GtkGrid over GtkBox.

But yes, there is a point. Having code easy for upcoming changes would probably be preferred…